projects/igniteui-angular/src/lib/core/styles/components/expansion-panel/_expansion-panel-theme.scss

```diff
@@ -114,10 +114,6 @@
         overflow: hidden;
     }
 
-    %igx-expansion-panel--expanded {
```

Comment:
@simeonoff 
I believe that we discussed this on one of the initial meetings - we will **not** be providing the margin / class out of the box, as it would be difficult for users to remove it, if they so wished.
@StefanIvanov 
There's a line in the [spec](https://github.com/IgniteUI/igniteui-angular/wiki/Accordion-Specification#3-functionality) that specifically mentions the margin of `8px` for expanded panels - can we remove it? It's easy enough to achieve in a sample by adding an `<igx-expansion-panel #panel [ngClass]="{'margin-class':  isPanelExpanded(panel) }">` and we'll make sure to showcase it in the docs.
We could add an `exportAs` to the `igx-expansion-panel`, so it's easier to get the panel state, but that's a separate enhancement.

Reply:
I remember we decided that the users will be setting this margin themselves. So, yes, removing this is the right thing to do.

Reply:
We want the default expanded state to be 8px apart from nearby panels, so I was thinking we would have an input for that that can be set to 0px if customers wanted to have no gaps like that.